stock_location_empty/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{'name': 'Stock Location Empty',
```

Comment:
Can you update this according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py ?